benchexec/tools/abc.py

```diff
@@ -24,7 +25,33 @@ def name(self):
         return "ABC"
 
     def cmdline(self, executable, options, task, rlimits):
-        return [executable] + options + [task.single_input_file]
+        return (
+            [executable]
+            + ["-c", "&r {}; &put".format(task.single_input_file)]
+            + options
+        )
+
+    def determine_result(self, run):
+        """
+        @return: status of ABC after executing a run
+        """
+        if run.was_timeout:
+            return "TIMEOUT"
+        status = None
+        for line in run.output:
```

Comment:
Is it by design that this loop does not return if a match is found and always checks all lines?

Same for avr.py.

Reply:
@nianzelee what is the intended behavior here?
Since we (usually) run only one analysis at a time. It does not matter if we return earlier or not.

Reply:
I think we can return when a match is found.

Reply:
Addressed in e31c8f0 and d0aa31a